scripts/linux/pkg/linux-packaging.md

```diff
@@ -0,0 +1,88 @@
+# RavenDB installation from package on Linux
+
+## What the package does on install
+
+- scatter the files in the default locations
+
+- install systemd service
+
+- add soft links to binaries
+
+- add `ravendb` user and group
+
+- set file permissions properly
+
+- setcap on the Raven.Server
+
+
+## Supported use cases
+
+### Ran under systemd
+The default is when RavenDB is used as a systemd daemon and controlled with systemd.
+The following directory layout (initialized by the package installation process) and permissions is used in this scenario:
+
+#### Filesystem locations and permissions
+
+- settings - `/etc/ravendb/settings.json`
+
+- data - `/var/lib/ravendb/data`
+
+- security (e.g. certificate)  - `/var/lib/ravendb/security`
```

Comment:
Who can read into this? Also, do we write the secret key into this location as well or to `$HOME/.secrets` ?

Reply:
Only `root` and `ravendb` group users. Also what determines where the `.secrets` is kept?

Reply:
It's either on `$HOME/.ravendb/secret.key` or you can specify `Security.MasterKey.Path` - I think we want the later option

Reply:
So I presume we want that like the rest of security - `/var/lib/ravendb/security/secret.key`.

Reply:
Yes

Reply:
You can specify that if it's there. If a new one needs to be generated it'll die with "no such file" https://github.com/ravendb/ravendb/blob/42bd9f403e176081040712826fb8e7b217b1afa7/src/Raven.Server/ServerWide/SecretProtection.cs#L726

But.. if it has length 0 it'll generate it. I'll just `touch` it in the script then.